.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+
+node_js:
+  - "10"
+
+before_install:
+  - curl https://yarnpkg.com/install.sh -sSfL | bash -s --
```

Comment:
Is it required to manually install yarn?

> If your Node.js project has the yarn.lock file in the repository root, we proceed to install Yarn and execute the yarn command instead of the default npm install.

- https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn

Reply:
Yes because otherwise Travis will use yarn 1.3.